.gitignore

```diff
@@ -1,6 +1,4 @@
 /target
-.idea/*
```

Comment:
You should leave this in here. I'm experimenting with idea RustRover

Reply:
hey how's that going? It seems great to me except that you can't connect to a vm over ssh

Reply:
I quite like it. I have a long history with intellij since my background is Java